internal/e2e/simple/simple_test.go

```diff
@@ -54,6 +64,10 @@ sh_binary(
 )
 -- subdir/subdir.sh --
 printf "Hello subdir!"
+-- test_simple_test_failing.sh --
+echo "1"
+-- test_simple_test_passing.sh --
+echo "0"
```

Comment:
Is there any way to have this test print out something to indicate that it is running (or not running) in coverage mode?

Reply:
Not sure. We would probably need a coverage report generator... Something like:
`bazel coverage --combined_report=lcov --coverage_report_generator=<an sh_library rule> <a cc_test rule>`
or maybe some other bazel targets to be able to add these arguments to `bazel coverage`:
`--coverage_output_generator (a build target label; default: "@bazel_tools//tools/test:lcov_merger")
  --coverage_report_generator (a build target label; default: "@bazel_tools//tools/test:coverage_report_generator")
  --coverage_support (a build target label; default: "@bazel_tools//tools/test:coverage_support")
  --[no]collect_code_coverage (a boolean; default: "false")`

Reply:
Let me know if you know a better approach.
I tested this implementation on a real project and it works perfectly.